include/mbedtls/check_config.h

```diff
@@ -203,6 +203,19 @@
 #endif
 #endif
 
+#if defined(MBEDTLS_PSA_CRYPTO_C) && \
```

Comment:
I think we should tell something about this in `mbedtls_config.h` as well, like "`MBEDTLS_CIPHER_C` if any unauthenticated cipher is enabled".

Reply:
Ah, thanks for fixing my mistake: it's about being builtin not just enabled.